### PR TITLE
Improve skew component

### PR DESCRIPTION
**Pull request by Xiaolei-Shawn**
## Description

- Remove numbers under skew bar and show them as tooltip on OI percentages
- Change height of table body row to 55px for trade history and futures market tables

## Related issue

https://github.com/Kwenta/kwenta/issues/1278

## Motivation and Context

Align to design

## Screenshots (if appropriate):
Screenshot 2022-08-23 at 22 53 00
Screenshot 2022-08-23 at 22 52 44

### CODE REVIEW

**Review by platschi — APPROVED** [review]
lgtm, design and functionality as expected 👍🏼

File: sections/futures/TradingHistory/TradesHistoryTable.tsx [hunk 1]
```diff
@@ -239,6 +239,7 @@ const StyledTable = styled(Table)<{ mobile?: boolean }>`
 	.table-body-row {
 		${TableAlignment}
 		padding: 0;
+		height: 55px;
```
Comment (Tburm) [inline sections/futures/TradingHistory/TradesHistoryTable.tsx:242]: Looks like this widened the row, but the content is no longer vertically centered:
image

**Review by Tburm — COMMENTED** [review]
Just one thing to fix on centering table rows

File: sections/futures/TradingHistory/TradesHistoryTable.tsx (hunk 1 above)
Comment (Tburm) [inline sections/futures/TradingHistory/TradesHistoryTable.tsx:242]:
```suggestion
```

File: sections/futures/FuturesMarketTabs/FuturesMarketTab.tsx [hunk 2]
```diff
@@ -192,6 +192,7 @@ const StyledTable = styled(Table)`
 	.table-body-row {
 		${TableAlignment}
 		padding: 6px 0px;
+		height: 55px;
```
Comment (Tburm) [inline sections/futures/FuturesMarketTabs/FuturesMarketTab.tsx:195]:
```suggestion
```

File: sections/futures/TradingHistory/SkewInfo.tsx [hunk 3]
```diff
@@ -72,32 +93,19 @@ const WithCursor = styled.div<{ cursor: 'help' }>`
...
+	p,
+	span {
+		font-size: 13px;
+		font-family: ${(props) =>
+			props.isNumber ? props.theme.fonts.mono : props.theme.fonts.regular};
+		color: ${(props) => props.theme.colors.selectedTheme.button.text};
+	}
 `;
 
 const SkewContainer = styled.div`
```
Comment (Tburm) [inline sections/futures/TradingHistory/SkewInfo.tsx:111]:
```suggestion
const SkewContainer = styled.div`
    height: 55px;
```

Comment (Tburm) [inline sections/futures/TradingHistory/SkewInfo.tsx:111]:
```suggestion
const SkewContainer = styled.div`
	height: 55px;
```

**Review by Tburm — APPROVED** [review]
@Xiaolei-Shawn sorry I blew up this PR but I fixed the thing I wanted to fix

**Pull request merged**